src/applications/personalization/profile/components/personal-health-care-contacts/Instructions.jsx

```diff
@@ -1,15 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import HelpDeskContact from '../HelpDeskContact';
+const Instructions = ({ testId, contactType }) => {
+  const isEmergency = contactType === 'emergency contact';
 
-const Instructions = ({ testId }) => (
-  <div data-testid={testId}>
-    To add a contact, call us at <HelpDeskContact testId={testId} />.
-  </div>
-);
+  return (
+    <div data-testid={testId}>
+      To add {isEmergency ? 'an' : 'a'} {contactType}, call your VA health
+      facility.{' '}
+      <href testId={testId} to="https://www.va.gov/find-locations/">
```

Comment:
Change to `<a />` tag, consider `href="/find-locations"`.

Reply:
thank you!